src/main/java/com/opentable/service/ServiceInfo.java

```diff
@@ -14,11 +14,151 @@
 
 package com.opentable.service;
 
+import java.net.URI;
+import java.util.Objects;
+
 /**
  * The {@link ServiceInfo} interface is intended for you to declare as a Spring Bean in your application.
  * It can be consumed by our components in order to create things like metric namespaces given information about your
  * service.  You are expected to implement your own implementations of this interface when defining your Bean.
+ * The easiest way to do so is use the Builder.
+ *
+ * For more details on these fields see https://wiki.otcorp.opentable.com/display/CP/The+service-details+Endpoint
  */
 public interface ServiceInfo {
+    /**
+     * Service Name
+     */
     String getName();
+
+    /**
+     * A description of the service
+     */
+    String description();
+
+    /**
+     * The name of the team responsible for the operation of the service.
+     */
+    String teamName();
+
+    /**
+     * The email of the team
+     */
+    String teamEmail();
```

Comment:
Why not have this be a type that performs some automatic validation on the value?  E.g., ensure the structure is: nonempty string, @, nonempty string, ., nonempty string.  (According to the RFC strictly is complicated and a waste of time.)

Reply:
Indeed, why not even require it ends with `@opentable.com`, or `@opentable.onmicrosoft.com`?

Reply:
Further, why not return a `List<Email>`?  Think about the Singularity owners.  .. Hm, although I guess the intent more here is explicitly for a team, and not individuals?